src/main/java/org/example/Pravokutnik.java

```diff
@@ -0,0 +1,16 @@
+package org.example;
+
+public class Pravokutnik extends GeometrijskiLik{
+    double stranicaA;
```

Comment:
Fieldovi u pravilu trebaju imati access modifier "private", jer ne želimo da ih netko izvan klase direktno mijenja. Možda želimo dodati neku kontrolu, recimo "ne može biti manji od nule", i onda to radimo u setter metodi u ovoj klasi. Ali ako field nije private, onda neka druga klasa može napraviti pravokutnik.stranicaA = -10. Ako je stranicaA private, onda to ne može napraviti.